src/components/atoms/status-indicator/index.tsx

```diff
@@ -0,0 +1,27 @@
+import clsx from "clsx"
+import React from "react"
+
+type StatusIndicatorProps = {
+  ok: boolean
+  okText: string
+  notOkText: string
+}
+
+const StatusIndicator: React.FC<StatusIndicatorProps> = ({
```

Comment:
We should consolidate this with the `fundamentals/status-dot` i like the name `status indicator` a lot better 😄